napari/layers/labels/labels.py

```diff
@@ -327,6 +335,12 @@ def mode(self, mode: Union[str, Mode]):
             self.interactive = False
             self.help = 'hold <space> to pan/zoom, drag to paint a label'
             self.mouse_drag_callbacks.append(paint)
+        elif mode == Mode.ERASE:
+            self.cursor_size = self.brush_size / self.scale_factor
+            self.cursor = 'square'
+            self.interactive = False
+            self.help = 'hold <space> to pan/zoom, drag to erase a label'
+            self.mouse_drag_callbacks.append(erase)
```

Comment:
```suggestion
            self.mouse_drag_callbacks.append(paint)
            self.selected_label = self.background_label
```
I think it is fine to have the selected label be the background label too. It removes the need for a separate highly redundant function in the mouse key bindings, unless i am missing something - in which case please let me know

Reply:
Given these suggestions I think we should also add something to the `selected_label` setter that checks whether someone has selected the background color, and if so change the mode into `erase`.

There is an additional problem now though, which I just thought of, which is that now someone can't use the background label with the fill bucket ...

Reply:
Think about that more we probably shouldn't auto change mode when `selected_label` is made equal to the background label, so people can still use fill as before